homeassistant/components/light/zwave.py

```diff
@@ -107,8 +107,8 @@ def update_properties(self):
 
     def _value_changed(self, value):
         """Called when a value has changed on the network."""
-        if self._value.value_id == value.value_id or \
-           self._value.node == value.node:
+        if self._value.value_id != value.value_id or \
+           self._value.node != value.node:
```

Comment:
I thought this might be related,  because it was how it was before I changed anything. But if it works without this of course no need for it to be so.